src/DocumentDB.ChangeFeedProcessor/DataAccess/ChangeFeedDocumentClient.cs

```diff
@@ -77,42 +78,70 @@ public async Task<IResourceResponse<DocumentCollection>> ReadDocumentCollectionA
         /// </summary>
         /// <param name="documentsFeedOrDatabaseLink">Collection Self Link.</param>
         /// <param name="document">Document to store.</param>
+        /// <param name="options">The <see cref="Microsoft.Azure.Documents.Client.RequestOptions"/>for this request.</param>
+        /// <param name="disableAutomaticIdGeneration">Disables the automatic id generation, If this is True the system will throw an exception if the id property is missing from the Document.</param>
```

Comment:
We can, but I would say since it's there in DocumentClient on this API, let's have it, i.e. for the APIs we use, let's not remove any parameters (for APIs that we don't use, just not add them). Let me know if you have strong opinion.